### PR TITLE
Add com.github.simonwhitaker.Gibo

### PR DESCRIPTION
**Pull request by saschpe**

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build com.github.simonwhitaker.Gibo

**Comment by saschpe** [comment]
Added *appdata.xml*, updated Git to 2.22.1 and removed some unused files ...

**Comment by saschpe** [comment]
bot, build com.github.simonwhitaker.Gibo

**Comment by saschpe** [comment]
bot, build com.github.simonwhitaker.Gibo

**Comment by saschpe** [comment]
bot, build com.github.simonwhitaker.Gibo

**Comment by saschpe** [comment]
bot, build com.github.simonwhitaker.Gibo

**Comment by saschpe** [comment]
bot, build com.github.simonwhitaker.Gibo

**Comment by saschpe** [comment]
bot, build com.github.simonwhitaker.Gibo

**Comment by saschpe** [comment]
:relieved:

**Comment by LeandroStanger** [comment]
bot, build com.github.simonwhitaker.Gibo

File: com.github.simonwhitaker.Gibo.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
```
Comment (gasinvein) [inline com.github.simonwhitaker.Gibo.appdata.xml]: Is this a command-line utility? If so, it's not `desktop-application`, but rather `console-application`.

Comment (saschpe) [reply com.github.simonwhitaker.Gibo.appdata.xml]: Done

**Comment by saschpe** [comment]
bot, build com.github.simonwhitaker.Gibo

**Comment by barthalion** [comment]
Have you contacted upstream author if they would be interested in maintainting their app on Flathub?

**Comment by saschpe** [comment]
It's quite possible, I'll ask him.
--
Mit freundlichen Grüßen,
Sascha Peilicke

File: com.github.simonwhitaker.Gibo.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+app-id: com.github.simonwhitaker.Gibo
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: gibo
+finish-args:
+  - --filesystem=~/.gitignore-boilerplates
```
Comment (bilelmoussaoui) [inline com.github.simonwhitaker.Gibo.yaml]: Any reasons for not using `$XDG_DATA_HOME` or `$XDG_CONFIG_HOME`?

File: com.github.simonwhitaker.Gibo.appdata.xml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
```
Comment (bilelmoussaoui) [inline com.github.simonwhitaker.Gibo.appdata.xml]: Should be upstreamed

File: com.github.simonwhitaker.Gibo.appdata.xml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  <id>com.github.simonwhitaker.Gibo</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Unlicense</project_license>
```
Comment (bilelmoussaoui) [inline com.github.simonwhitaker.Gibo.appdata.xml]: Is the maintainer aware of this?

Comment (saschpe) [reply com.github.simonwhitaker.Gibo.appdata.xml]: Yes, https://github.com/simonwhitaker/gibo/blob/master/UNLICENSE

File: com.github.simonwhitaker.Gibo.yaml (hunk 2 above)
Comment (saschpe) [reply com.github.simonwhitaker.Gibo.yaml]: That's where the tool puts its stuff. There is no way to influence that.

File: com.github.simonwhitaker.Gibo.appdata.xml (hunk 3 above)
Comment (saschpe) [reply com.github.simonwhitaker.Gibo.appdata.xml]: https://github.com/simonwhitaker/gibo/pull/44

File: com.github.simonwhitaker.Gibo.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
+app-id: com.github.simonwhitaker.Gibo
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
```
Comment (LeandroStanger) [inline com.github.simonwhitaker.Gibo.yaml]: Update runtime to 19.08

Comment (nedrichards) [reply com.github.simonwhitaker.Gibo.yaml]: This is not specifically required, but would be good to check.

Comment (saschpe) [reply com.github.simonwhitaker.Gibo.yaml]: Done, works as well

**Comment by saschpe** [comment]
bot, build com.github.simonwhitaker.Gibo

File: README.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
+# Gibo
+[gibo][gibo] (short for .gitignore boilerplates) is a shell script to help you easily
+access .gitignore boilerplates from [Github.com][gitignore].
+
+## Build and install
+Assuming *flatpak* and *flatpak-builder* are installed, follow the next steps:
+
+```bash
+$ git clone https://github.com/flathub/com.github.simonwhitaker.Gibo.git
```
Comment (bilelmoussaoui) [inline README.md]: Please remove those commands from here. The docs has already this.

Comment (saschpe) [reply README.md]: Not quite sure what you mean, that isn't part of the diff against `new-pr`

**Comment by saschpe** [comment]
Any update?

**Comment by nedrichards** [comment]
> Any update?

What did upstream say about having commit access? Other than that, this seems fine.

**Comment by nedrichards** [comment]
bot, build com.github.simonwhitaker.Gibo

**Comment by saschpe** [comment]
@nedrichards Are you referencing to @barthalion 's question?
> Have you contacted upstream author if they would be interested in maintainting their app on Flathub?

`AppData.xml` was merged upstream so I guess I can remove it once Simon publishes a new release tarball.

**Comment by nedrichards** [comment]
> @nedrichards Are you referencing to @barthalion 's question?
> 
> > Have you contacted upstream author if they would be interested in maintainting their app on Flathub?

Yep, that's exactly it. This request has the 'awaiting upstream' tag on it which means we're awaiting an indication of whether the upstream software authors want to be involved in making the app available. There's some more colour around this on the App Submissions page https://github.com/flathub/flathub/wiki/App-Submission#someone-else-has-put-my-app-on-flathubwhat-do-i-do

If they don't it's not an absolute blocker, but we want to`make sure we're asking first.
 
> `AppData.xml` was merged upstream so I guess I can remove it once Simon publishes a new release tarball.

Perfect, yes! This is one of the reasons we're so keen to have upstream involved (or if not, to make sure that the people who work on things on flathub become part of upstream)

**Comment by nedrichards** [comment]
bot, build com.github.simonwhitaker.Gibo

**Pull request closed without merging**